src/main/java/bio/terra/app/utils/SentryUtils.java

```diff
@@ -0,0 +1,46 @@
+package bio.terra.app.utils;
+
+import bio.terra.app.configuration.SentryConfiguration;
+import bio.terra.common.exception.BadRequestException;
+import bio.terra.common.exception.ConflictException;
+import bio.terra.common.exception.ForbiddenException;
+import bio.terra.common.exception.NotFoundException;
+import bio.terra.common.exception.NotImplementedException;
+import io.sentry.Sentry;
+import java.util.Optional;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.servlet.NoHandlerFoundException;
+
+public class SentryUtils {
+  private static final String DEFAULT_UNDEFINED_ENVIRONMENT = "undefined";
+
+  public static void initializeSentry(SentryConfiguration sentryConfiguration) {
```

Comment:
Instead of adding a call to this method from `JobService`, the method itself could be added as a `PostConstruct` handler to `SentryConfiguration`. Using `JobService`, which don't have anything to do with Sentry, seems like the wrong place to add it.

Reply:
Oh, cool! This is a much better approach, thank you!